### ISSUE TITLE
Suddenly crashes shortly after starting

### ISSUE THREAD

**Issue by JoeTN** [haugene/docker-transmission-openvpn#929]
First of all thanks for your great work!

I have been a happy user of your image for some time now. 
Suddenly my container is crashing without further notice. I haven't changed anything, so I have no idea what is causing this. 

I can start the container and it runs perfectly fine anywhere between 1 to 5 minutes. When it runs I can download torrents just fine, suddenly it crashes. I can start it again, and the same is happening. As stated it used to run just perfect all the time. Can't seem to find any relevant in the logs... There's probably more logs that I aren't ware of ?

I've tried both 2.5, 2.6 and latest. The problem is the same.

My docker compose file:
[https://pastebin.com/madns1JB](url)

Container log:
[container-log.txt](https://github.com/haugene/docker-transmission-openvpn/files/3752173/container-log.txt)

Transmission log:
[transmission.log](https://github.com/haugene/docker-transmission-openvpn/files/3752194/transmission.log)

My system:
QNAP TS451+
Container Station V2.0.356
Docker version 17.09.1-ce, build e398b97

Best regards
Joe

**Comment by n2go** [comment]
Did you ever find a solution to this?
I noticed just now that mine is also no longer working after having worked fine for a long time.

It crashes after a few seconds in my case. 

TS-251+
version 4.4.1.1146
Latest image version.

**Comment by JoeTN** [comment]
Sadly I haven't found a solution to the problem yet... Please let me know if you figure it out!

**Comment by n2go** [comment]
well i gave up, i have moved to deluge vpn docker... (doesnt crash)

**Comment by haugene** [comment]
Does it still crash if you don't specify the LOCAL_NETWORK env var?
From the logs posted I see that the `192.168.10.0/24` route add command gives an error.

Without it you won't be able to access the webui without a proxy, but I'm interested to see if it then runs without crashes.

**Comment by JoeTN** [comment]
Actually I got back to start my container the other day to try your suggestion @haugene, started it ant noticed that it did'nt crash as it used to do... It have been running solid for days now. I don't know what have changed in the meantime from when I first described the problem here to now where it has fixed itself... Not other than I've upgraded the OS on my QNAP a couple of times in between, so something must have solved it there.

Working for now and I'm crossing my fingers. Happy new year.
